.gitignore

```diff
@@ -0,0 +1 @@
+go-blueprint
```

Comment:
Why are we ignoring this? Is this the binary after `go build` ?

Reply:
To avoid seeing it when doing a 
git status

Also, not to mistakenly add it to the repo on a commit.

It is not important. Would you want me to remove it, or should we keep it?

Reply:
yes it is the binary after go build